test.md

```diff
@@ -0,0 +1,2 @@
+## this is read me file
+# this file is not upload
```

Comment:
- [ ] As mentioned by the previous reviewer, the `test.md` file is meant to be ignore inside `.gitignore` file. But due to the fact that, you've committed and push it for the first time, you might have some issues removing it from your repo. Kindly follow this [link](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) to address the issue. I wish you the best :handshake: 

![testmd](https://user-images.githubusercontent.com/26597432/219014925-23b8f0cf-893c-416c-9915-7d62d2c4c11f.png)